app/tab_deep_dive_countries.py

```diff
@@ -84,8 +84,24 @@ def content_deep_dive_countries(api: PtxboaAPI, res_costs: pd.DataFrame) -> None
         x = None
         st.markdown("TODO: fix surplus countries in data table")
 
-    c1, c2 = st.columns(2, gap="medium")
-    with c2:
+    st.subheader("Full load hours of renewable generation")
+    c1, c2 = st.columns([2, 1], gap="large")
+    with c1:
+        st.markdown("**Map of input data values**")
+        if data_selection in ["full load hours", "CAPEX"]:
+            map_parameter = st.selectbox(
+                "Show Parameter on Map", process_code, key="ddc_flh_map_parameter"
+            )
+        else:
+            map_parameter = "interest rate"
+        fig = plot_input_data_on_map(
+            api=api,
+            data_type=data_selection,
+            color_col=map_parameter,
+            scope=ddc,
+        )
+        st.plotly_chart(fig, use_container_width=True)
+    with st.expander(f"**Region specific {data_selection} data**"):

```

Comment:
Just use ``Data:`` as header of the expander and remove the markdown header below

Reply:
okay, I thought this is more explicit but don't have a strong opinion.